multi_import/multi_importer.py

```diff
@@ -85,9 +85,20 @@ def import_data(self, data):
 
         bound_importers = self._transform_multi_input(data)
 
-        for importer, filename, dataset in bound_importers:
-            result = importer.import_data(dataset, context, transaction=False)
-            results.add_result(filename, result)
+        for importer, datasets in bound_importers:
```

Comment:
I think we're missing `filename` here.  this line should be the same as it was, I think.  Yea?